src/domains/ioweb-common/10_spid_login.tf

```diff
@@ -112,7 +112,7 @@ module "spid_login" {
     # Spid logs
     ENABLE_SPID_ACCESS_LOGS             = true
     SPID_LOGS_ENABLE_PAYLOAD_ENCRYPTION = false
-    SPID_LOGS_STORAGE_CONNECTION_STRING = module.spid_logs_storage.primary_connection_string
+    SPID_LOGS_STORAGE_CONNECTION_STRING = module.immutable_spid_logs_storage.primary_connection_string
```

Comment:
What if we change the pointing to the resources on another PR?

Reply:
Done in [fix based on comments](https://github.com/pagopa/io-infra/pull/778/commits/1e4268778c5a0b35cfe764917907553e84ffb6de)